### ISSUE TITLE
[stable/airflow] Existing Secret Configuration Not working

### ISSUE THREAD

**Issue by zachbolian** [helm/charts#14142]
**Describe the bug**
When setting the `existingAirflowSecret` and `airflow.secretsMapping` properties in the values.yaml file those configurations are not pulled onto the web, worker, scheduler, etc containers. 

**Version of Helm and Kubernetes**:
Helm: `v2.14.0`
Kubes: `1.11.8-gke.6`

**Which chart**:
`stable/airflow`

**What happened**:
When setting the `existingAirflowSecret` and `airflow.secretsMapping` properties in the values.yaml the container set environment variables based on the `defaultSecretsMapping`

**What you expected to happen**:
When setting the `existingAirflowSecret` and `airflow.secretsMapping` properties in the values.yaml the container set environment variables based on the `airflow.secretsMapping`

**How to reproduce it** (as minimally and precisely as possible):
values.yaml:
```
airflow:
  secretsMapping:
  - envVar: POSTGRES_USER
    secretName: airflow-creds
    secretKey: username
  - envVar: POSTGRES_PASSWORD
    secretName: airflow-creds
    secretKey: password
  - envVar: REDIS_PASSWORD
    secretName: airflow-creds
    secretKey: password

existingAirflowSecret: airflow-creds
```

`helm template -f values.yaml -n airflow stable/airflow -x templates/deployments-web.yaml`

Inspect under `containers.env` and instead of the secret `airflow-creds` being used you will see the default `airflow-env`

**Anything else we need to know**:
I believe this is related to how variables are scoped in helm templates. Specifically, I think the updates to the `$mapping` and `$secretName` done here https://github.com/helm/charts/blob/master/stable/airflow/templates/_helpers.tpl#L89-L93 are not respected once you are outside the if conditions where the variables are updated.

**Comment by JonnyIncognito** [comment]
This is fixed in PR #11820

**Comment by zachbolian** [comment]
Ah, thanks! Had searched the issues but not the PRs. Closing.
